README.md

```diff
@@ -33,8 +33,8 @@ feature-flags.
 
 Afterwards, type `nix flake show` in the root directory of the project to see
 all the available versions of this book. Then type `nix build .#<edition>` to
-build the edition you want (Haskell, Scala, OCaml, Reason and their printed
-versions). For example, to build the Scala edition you'll have to type
+build the edition you want (Haskell, Scala, OCaml, Reason, Agda and their
+printed versions). For example, to build the Scala edition you'll have to type
```

Comment:
I think, after inserting "Agda," Prettier didn't like how long line 36 was, so I moved the last word ("printed") to the next line.

Reply:
Thank you